### DISCUSSION TITLE
Reduced time investment due to Unity's recent actions

### DISCUSSION THREAD

**Discussion by simonoliver** [simonoliver/UnityFigmaBridge#35] [Announcements]
Given Unity's [recent announcement](https://blog.unity.com/news/plan-pricing-and-packaging-updates) - I'm pausing work on all my open source Unity packages. I feel this represents a huge breach of trust and won't be investing any more time in the platform unless they roll back these changes. 

(UPDATE) - I've spent a lot of time reflecting on this after their update. While I do feel it's an improvement, I don't have faith in the leadership of the company to provide the environment that Unity needs to succeed going forward and I don't have faith that they aren't going to do something similar in another 6 months time. As a result, I'm going to be shifting my efforts to Unreal Engine and hope to create similar open-source packages for UE (and potentially Godot). I'll still be maintaining this package, but won't be investing significant chunks of time on new features.

**Comment by siamonas** [comment]
Hopefully, they will reconsider...

[Recent tweet from Unity's account](https://x.com/unity/status/1703547752205218265?s=20)
